app/javascript/topics/components/TopicsShow__PostEditor.re

```diff
@@ -199,6 +201,7 @@ let make =
       }
     );
   let updateMarkdownCB = body => setState(state => {...state, body});
+  let (editReason, setEditReason) = React.useState(() => None);
```

Comment:
This component already has a record-type `state`. Why not just add the `reason` key to it, and use `state` and `setState`? 🤔